core/src/main/java/pl/agh/edu/actor/slider/SliderComponent.java

```diff
@@ -0,0 +1,69 @@
+package pl.agh.edu.actor.slider;
+
+import java.util.function.Function;
+
+import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.utils.Align;
+
+import pl.agh.edu.actor.HotelSkin;
+
+public class SliderComponent extends Table {
+	private int SLIDER_WIDTH = 600;
+	private int HORIZONTAL_PADDING = 20;
+	private Label valueLabel;
+	private Slider slider;
+	private Skin skin;
+	private String suffix;
+	private Function<Float, Float> mapping;
+
+	public SliderComponent(String name, String suffix, float min, float max, float step, Function<Float, Float> mapping) {
```

Comment:
suffix raczej powinien być u standaryzowany to wszystkich

Reply:
tzn? Będzie jakaś funkcja która będzie podstawiać np. $ do wartości pieniężnych?